system/jlib/jcontainerized.cpp

```diff
@@ -231,8 +231,12 @@ bool applyYaml(const char *componentName, const char *wuid, const char *job, con
     const char *runtimeImageVersion = baseImageVersion; // runtime image version will equal base version unless changed dynamically below
     for (const auto &p: extraParams)
     {
+        if (streq(p.first.c_str(), "instanceNum"))
```

Comment:
missed this on last review.. why treat specially?
i.e. afaics/r, it is a 'normal' param, and without this special casing will be added by line 259 in same way.